src/ComparatorByDefectCount.java

```diff
@@ -0,0 +1,8 @@
+import java.util.Comparator;
+
+public class ComparatorByDefectCount implements Comparator<Vehicle> {
+    @Override
+    public int compare(Vehicle o1, Vehicle o2) {
+        return (MechanicService.getSumNumberOfBreaks(o2)) - (MechanicService.getSumNumberOfBreaks(o1));
```

Comment:
it's should be in similar style
in another class you comparing by Double.compare
maybe hear also better to compare by Integer.compare or something like that

Reply:
fixed